components/common/ReviewScore.tsx

```diff
@@ -12,7 +12,7 @@ export const ReviewScore = ({
   const ratingValue = (rating / 5) * 100;
   return (
     <Flex w="100%" justifyContent="space-between">
-      <Text>{name}</Text>
+      <Text>{name[0].toUpperCase() + name.slice(1)}</Text>
```

Comment:
you could do soemthing likes this will fix the other case
```js
const modifiedReview = review.replace(/([A-Z])/g, " $1");
const reviewString = result.charAt(0).toUpperCase() + result.slice(1);
```

Reply:
Genius! Implemented the changes in my latest commit.